src/component/helper/RoamController.ts

```diff
@@ -179,6 +179,14 @@ class RoamController extends Eventful<{
             return;
         }
 
+        let el = e.target;
+        while (el) {
+            if (el.draggable) {
+                return;
+            }
+            el = el.__hostTarget ? el.__hostTarget : el.parent;
```

Comment:
It can be written as:
```ts
el = el.__hostTarget || el.parent;
```

Also, I'm not sure about what's the scene of this logic and if it's correct.  `e.target.draggable` has been checked and return earlier.

Reply:
When dragging el's textContent, it will also trigger the pan effect which is unexpected. So I check if its parent is draggable.

Reply:
Understand, https://github.com/apache/echarts/blob/472e72b6fde58a2ff9f14f698e3c197cb03ec917/src/component/helper/RoamController.ts#L177 this line of code can be removed.